### ISSUE TITLE
WCAG 2.2: “New Features in WCAG 2.2” has different formatting than WCAG 2.1 equivalent

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#1807]
The formatting in the [“New Features in WCAG 2.2“ section in the WCAG 2.2](https://www.w3.org/TR/WCAG22/#new-features-in-wcag-2-2) WD is different from the formatting in [“New Features in WCAG 2.1” in WCAG 2.1](https://www.w3.org/TR/WCAG21/#new-features-in-wcag-2-1).

* The new Success Criteria are randomly ordered, they should be ordered by SC number, as it was in WCAG 2.1.
* The new Success Criteria do not name the SC number. This should be included, as it was in WCAG 2.1.
* The new Success Criteria do not name the level of the SC. This should be included, as it was in WCAG 2.1.

The /TR/ documents are hard to read as they are, let’s lower the cognitive load for people to format them comparably.

**Comment by yatil** [comment]
@michael-n-cooper @alastc Seeing that there is a CfC for WCAG 2.2 going to candidate rec, I'd like to bump this issue. (It looks like it was never properly tagged and or considered. Happy with a “nah, won’t fix”, if that’s what you feel.)

**Comment by shawna-slh** [comment]
+1

Also, please add a link prominently at the top to [What's New in WCAG 2.2 Working Draft](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/), which we will update with each publication. Folks really appreciate that concise list of the new SCs and they find those persona quotes help with understanding.

**Comment by yatil** [comment]
I put in PR #2667 for my issue. I don’t know where to best put a link to “What’s New in WCAG 2.2”. I leave that to a different issue/Pull Request.

**Comment by yatil** [comment]
>  CR has a [New Features in WCAG 2.2](https://www.w3.org/TR/WCAG22/#new-features-in-wcag-2-2) similar to 2.1 but @yatil am I not looking closely enough?

No SC numbers, no levels, see the pull request preview and compare.

**Comment by bruce-usab** [comment]
Okay, I see.  I thought the alphabetical sort was okay (since it is a smaller listing than 2.1 had) but adding SS numbers and levels would be good.  Certainly we should settle on best formatting for TR.

**Comment by yatil** [comment]
Done in https://github.com/w3c/wcag/pull/2780

**Comment by bruce-usab** [comment]
@shawna-slh --

> Also, please add a link prominently at the top to [What's New in WCAG 2.2 Working Draft](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/), which we will update with each publication.

Is this in done by 2780 or otherwise in the current editors draft?  If not, does this need to be a new issue?

**Comment by shawna-slh** [comment]
Thanks for the catch, @bruce-usab ! Looks like not. I added https://github.com/w3c/wcag/pull/2843
